subprojects/docs/src/docs/userguide/dep-man/01-core-dependency-management/declaring_repositories.adoc

```diff
@@ -769,7 +769,8 @@ e.g. `gcs://myBucket/maven/release`
 == Handling credentials
 
 Repository credentials should never be part of your build script but rather be kept external.
-Gradle provides link:{javadocPath}/org/gradle/api/artifacts/repositories/AuthenticationSupported.html#credentials-java.lang.Class-[an API in artifact repositories]
+For link:{javadocPath}/org/gradle/api/credentials/PasswordCredentials.html[PasswordCredentials]
```

Comment:
I would make it more visible that this is limited to a set of credentials, by using a `NOTE` like above around line 725